### PR TITLE
Deal with no data

### PR DESCRIPTION
**Pull request by vondele**

### CODE REVIEW

**Comment by vondele** [comment]
useful for exploring WDL models for individual commits, not every commit has data available.

File: scoreWDL.py [hunk 1]
```diff
@@ -800,6 +802,10 @@ def create_plot(self, model_data_density: ModelDataDensity, model: ModelData):
         args.yData,
     )
 
+    if len(win) + len(draw) + len(loss) == 0:
```
Comment (robertnurnberg) [inline scoreWDL.py:805]: later on we skip mom values with `len(ywindata) < 10`, so in theory we could bail out here already if `len(win)<100` or similar. but not sure

**Comment by robertnurnberg** [comment]
how about adding your script to obtain the per-commit raw data of the normalizing constant to the repo?

File: scoreWDL.py (hunk 1 above)
Comment (vondele) [reply scoreWDL.py:805]: I think it is better to deal with that separately. Also have in total just few data points is a bit unlikely, even parsing a single pgn would leads to 10s of data points.

**Comment by vondele** [comment]
I'll make a separate PR, but probably just to share and not to merge at this point, since it is a bit of a hack.

**Comment by robertnurnberg** [comment]
lgtm

**Pull request merged**